### PR TITLE
WIP - chore: Support force pushing a tag

### PR DESCRIPTION
**Pull request by daveconde**
Fixes: #6071 and also adds support for updating an existing PR during `jx create pr` For example: `jx create pr -l my-pr --update-existing=true`

### CODE REVIEW

File: pkg/cmd/create/create_pullrequest.go [hunk 1]
```diff
@@ -86,6 +87,7 @@ func NewCmdCreatePullRequest(commonOpts *opts.CommonOptions) *cobra.Command {
 	cmd.Flags().StringArrayVarP(&options.Labels, "label", "l", []string{}, "The labels to add to the pullrequest")
 	cmd.Flags().BoolVarP(&options.Push, "push", "", false, "If true the contents of the source directory will be committed, pushed, and used to create the pull request")
 	cmd.Flags().BoolVarP(&options.Fork, "fork", "", false, "If true, and the username configured to push the repo is different from the org name a PR is being created against, assume that this is a fork")
+	cmd.Flags().BoolVarP(&options.UpdateExisting, "update-existing", "", false, "If true, lookup existing PRs based on the labels passed in and update an existing PR if one is found")
```
Comment (hferentschik) [inline pkg/cmd/create/create_pullrequest.go]: How is this related to "force pushing" a tag? +1 for being able to update an existing PR, but how is this related to force pushing.

File: pkg/cmd/step/step_tag.go [hunk 2]
```diff
@@ -94,7 +95,7 @@ func NewCmdStepTag(commonOpts *opts.CommonOptions) *cobra.Command {
 	cmd.Flags().StringVarP(&options.Flags.ChartValueRepository, "charts-value-repository", "r", "", "the fully qualified image name without the version tag. e.g. 'dockerregistry/myorg/myapp'")
 
 	cmd.Flags().BoolVarP(&options.Flags.NoApply, "no-apply", "", false, "Do not push the tag to the server, this is used for example in dry runs")
-
+	cmd.Flags().BoolVarP(&options.Flags.ForcePush, "force-push", "", false, "force push the tag to the server")
```
Comment (hferentschik) [inline pkg/cmd/step/step_tag.go]: I am a bit concerned around force pushing tags. Where do you want to use this? What if someone else already has received the old tags? Will he not get errors/warnings when pulling the next time?

Comment (hferentschik) [reply pkg/cmd/step/step_tag.go]: What is the underlying problem we are trying to solve?

File: pkg/cmd/create/create_pullrequest.go (hunk 1 above)
Comment (daveconde) [reply pkg/cmd/create/create_pullrequest.go]: I'll create another issue to capture the pull request not updating
